hal/common/retarget.cpp

```diff
@@ -98,6 +98,9 @@ static void init_serial() {
 #if DEVICE_SERIAL
     if (stdio_uart_inited) return;
     serial_init(&stdio_uart, STDIO_UART_TX, STDIO_UART_RX);
+#if MBED_CONF_CORE_STDIO_BAUD_RATE
```

Comment:
What is the default baud if the config isnt present? Seems there should have been a defined value that is assigned the baud rate rather than conditionally making the set baud call? Maybe Ive missed a subtly.